app/io.github.niccokunzmann.python_dhcp_server.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.niccokunzmann.python_dhcp_server</id>
+
+  <name>Python DHCP Server</name>
+  <summary>Serve and inspect local IPs</summary>
+
+  <developer id="eu.quelltext">
+    <name >Nicco Kunzmann</name>
+  </developer>
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+
+<!--
+  See https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#oars-information
+  Generated using https://hughsie.github.io/oars/generate.html
+-->
+  <content_rating type="oars-1.0" />
+
+<!--
+  <icon type="stock">eu.quelltext.open_chakra_toning</icon>
+  <icon type="remote">https://raw.githubusercontent.com/niccokunzmann/open_chakra_toning/main/assets/img/icon/icon.svg</icon>
+-->
+  <url type="bugtracker">https://github.com/niccokunzmann/python_dhcp_server/issues</url>
+  <url type="homepage">https://github.com/niccokunzmann/python_dhcp_server/</url>
+  <url type="donation">https://github.com/niccokunzmann/python_dhcp_server/</url>
+  <url type="contact">https://github.com/niccokunzmann/python_dhcp_server/issues</url>
+  <url type="faq">https://github.com/niccokunzmann/python_dhcp_server/</url>
+  <url type="translate">https://github.com/niccokunzmann/python_dhcp_server/</url>
+  <url type="vcs-browser">https://github.com/niccokunzmann/python_dhcp_server/</url>
+
+  <description>
+    <p>
+      This DHCP server program will assign IP addresses ten seconds after it received packets from clients. So it can be used in networks that already have a dhcp server running. For most (WIFI) networks, it will just show what is happening and not interfere.
+    </p>
+    <p>
+      This Python DHCP server
+    </p>
+    <ul>
+      <li>shows clients in the network</li>
+      <li>lists IP address, Mac address and host name</li>
+      <li>highlights recently refreshed/added clients</li>
+      <li>assigns IP addresses 10 seconds later than usual DHCP servers</li>
+      <li>remembers addresses in the hosts.csv file.</li>
+      <li>can be configured to serve all DHCP options using Python</li>
+    </ul>
+    <p>
+      Contributions welcome!
+    </p>
+    <p>
+      Installation and running need to be done as root because the DHCP port
+      can not be accessed as a normal user:
+    </p>
+    <ol>
+      <li>sudo flatpak install io.github.niccokunzmann.python_dhcp_server</li>
+      <li>sudo -i flatpak run io.github.niccokunzmann.python_dhcp_server</li>
+    </ol>
+    <p>
+      If you wish, you can edit the .desktop file, so you can run the program as root.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">io.github.niccokunzmann.python_dhcp_server.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/niccokunzmann/python_dhcp_server/master/images/dhcpgui.png</image>
```

Comment:
this should link to a specific commit otherwise it might just disappear.

Reply:
also this is a Windows screenshot.

Reply:
Oh, right ... updating...

Reply:
This is now moved into upstream and I use the hash.. Thanks for finding this!